crates/sail-plan/src/function/scalar/array.rs

```diff
@@ -2,6 +2,7 @@ use arrow::datatypes::DataType;
 use datafusion::functions_nested::expr_fn;
 use datafusion_common::ScalarValue;
 use datafusion_expr::{expr, lit, BinaryExpr, Operator};
+use datafusion_functions_nested::position::array_position_udf;
```

Comment:
We have `use datafusion_functions_nested` and `use datafusion::functions_nested`. I'm not sure if there is a way to prefer one over the other in the linter.